CHANGELOG.md

```diff
@@ -37,6 +37,13 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - [bridge/ot] Fall-back to TextMap carrier when it's not ot.HttpHeaders. (#3679)
 - The `Collect` method of the `"go.opentelemetry.io/otel/sdk/metric".Reader` interface is updated to accept the `metricdata.ResourceMetrics` value the collection will be made into. This change is made to enable memory reuse by SDK users. (#3732)
+- The structure of the `go.opentelemetry.io/otel/metric/unit` package is updated. (#3761)
```

Comment:
If we are thinking about long-term, do units belong within the metric package?  Is it possible other signals will want to incorporate units at some point?

Reply:
That's a good question. I don't know of other places this would be used, but I don't know what I don't know either.

I can see in the case another signal needs this package that we do something similar to what we plan for the `metric/global` package: move it to the top level `otel` package.

Reply:
I'm fine with during the RC cycle deprecating this `otel/metric/unit` package and copying it to `otel/unit`. The post-RC release would remove this package.

@open-telemetry/go-maintainers thoughts?